package.json

```diff
@@ -195,6 +195,18 @@
     "zen-observable": "^0.8.15",
     "zod": "^3.22.2"
   },
+  "peerDependencies": {
+    "chromatic-playwright": "^0.3.1",
+    "chromatic-cypress": "^0.3.2"
```

Comment:
If we're going to cut 1.0.0 versions of these when we go GA, we might want to include that here.

Reply:
OK, we can do that. It'll still be possible to test (with incorrect versions) against the current versions if we do that, but it'll warn in some package managers, and error in npm: https://www.notion.so/chromatic-ui/Peer-Dependency-Investigations-ae77792bae6e420bbe860916bec10cbe?pvs=4#176332d500194cdfacadf1efd927c815

Reply:
Oh, I misinterpreted, you mean to have both versions supported 👍